src/ChatGptNet/ChatGptClient.cs

```diff
@@ -15,7 +15,7 @@ internal class ChatGptClient : IChatGptClient
     private readonly IMemoryCache cache;
     private readonly ChatGptOptions options;
 
-    private static readonly JsonSerializerOptions jsonSerializerOptions = new(JsonSerializerDefaults.Web);
+    private static readonly JsonSerializerOptions jsonSerializerOptions = new(JsonSerializerDefaults.Web) { DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull };
```

Comment:
Fix the formatting: braces should be on a new line, as the property (see other object initializations in the file).